### PR TITLE
Add default error page

### PR DESCRIPTION
**Pull request by brunobar79**
Fixes APP-####

## Screen recordings / screenshots

https://github.com/rainbow-me/rainbow/assets/1247834/7ea1c0f2-a60a-4025-85ec-43919432a8e4

### CODE REVIEW

File: src/components/DappBrowser/ErrorPage.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import lang from 'i18n-js';
+import React from 'react';
+import { Centered } from '../layout';
+import { Text } from '../text';
```
Comment (skylarbarrera) [inline src/components/DappBrowser/ErrorPage.tsx:4]: you're playing with fire

**Review by skylarbarrera — APPROVED** [review]

File: src/components/DappBrowser/ErrorPage.tsx (hunk 1 above)
Comment (brunobar79) [reply src/components/DappBrowser/ErrorPage.tsx:4]: I heard you're great at making new screens

**Review by benisgold — APPROVED** [review]
it's beautiful. my 1 nit pick is that when this screen is shown is it ever actually possible that reloading the page as suggested will fix the problem?

**Comment by brunobar79** [comment]
> it's beautiful. my 1 nit pick is that when this screen is shown is it ever actually possible that reloading the page as suggested will fix the problem?

That's a good point. For most errors it won't be recoverable so the current copy isn't ideal. Do you have any suggestion?

File: src/components/DappBrowser/ErrorPage.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import lang from 'i18n-js';
```
Comment (walmat) [inline src/components/DappBrowser/ErrorPage.tsx:1]:
```suggestion
import * as lang from '@/languages';
```

File: src/components/DappBrowser/ErrorPage.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        backgroundColor: colors.white,
+      }}
+    >
+      <Box alignItems="center" padding="20px">
+        <Text letterSpacing="zero" size="h2">
+          {`🫠`}
+        </Text>
+        <Centered marginTop={9}>
+          <Text color={colors.dark} size="big" weight="bold">
+            {lang.t('dapp_browser.error.title')}
```
Comment (walmat) [inline src/components/DappBrowser/ErrorPage.tsx:43]:
```suggestion
            {lang.t(lang.l.dapp_browser.error.title)}
```

File: src/components/DappBrowser/ErrorPage.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+          </Text>
+        </Centered>
+        <Box padding={'12px'}>
+          <BodyText color={colors.dark}>{lang.t('dapp_browser.error.default_msg')}</BodyText>
```
Comment (walmat) [inline src/components/DappBrowser/ErrorPage.tsx:47]:
```suggestion
          <BodyText color={colors.dark}>{lang.t(lang.l.dapp_browser.error.default_msg)}</BodyText>
```

File: src/components/DappBrowser/ErrorPage.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+const BodyText = styled(Text).attrs(({ theme: { colors } }: { theme: { colors: Colors } }) => ({
+  align: 'center',
+  color: colors.alpha(colors.blueGreyDark, 0.6),
+  lineHeight: 'loosest',
+  size: 'medium',
+}))({
+  paddingBottom: 23,
+  paddingTop: 4,
+});
```
Comment (walmat) [inline src/components/DappBrowser/ErrorPage.tsx:19]: Possible to use the `Text` import from `@/design-system` instead here?

Comment (brunobar79) [reply src/components/DappBrowser/ErrorPage.tsx:19]: This is just a placeholder. Will do once we get designs

**Pull request merged**